### ISSUE TITLE
Desktop environment please

### ISSUE THREAD

**Issue by manhbh** [lowspecman420/AnBox86#18]
Can you integrate the desktop environment and XRDP or tigerVNC into Anbox64-86.  I tried to install but no success.  Thank you very much

**Comment by nakele** [comment]
I installed TigerVNC and XFCE4 or LXDE and they both work via TigerVNC. Of course you need to do some configurations and some changes. It also works with XServer android app or the X11 Termux apk.

Setup isn't any different from what you find in online tutorials, this will give you the basics covered: https://www.youtube.com/watch?v=TDPwxibFG5E

One more things, if you install now, there some bits that dont work so you will have to compile some bits yourself, see this for reference: https://github.com/lowspecman420/AnBox86/issues/17

**Comment by manhbh** [comment]
@nakele  Thank you so much. But the wine can't run via the desktop environment. How can I run it in the XFCE?

**Comment by nakele** [comment]
> @nakele Thank you so much. But the wine can't run via the desktop environment. How can I run it in the XFCE?

What the terminal say when you launch wine? Does it say box86 is not a command?

If so you need to compile box86 by yourself at the moment until this project gets some fixes: see #17 in my previous comment.

**Comment by manhbh** [comment]
I want to run *.exe file and access via VNC viewer. The terminal say like this 
![IMG_20220727_183021](https://user-images.githubusercontent.com/93500972/181236600-45f31372-16f1-4412-a368-01dbb36463d8.jpg)

**Comment by nakele** [comment]
@manhbh see the first error in the console? box86: command not found. You need to compile box86 yourself until this project finds a way around it. See #17

**Comment by manhbh** [comment]
I follow line 71 in AnBox86_64.sh by manual and I get this error. Look like the link died already.
![5a04787486bf44e11dae](https://user-images.githubusercontent.com/93500972/181266122-755d6bf2-d197-4bfc-aa21-54019e9cc834.jpg)

**Comment by nakele** [comment]
@manhbh yeah, the problem is the link. You need to go to the box86 repo and compile yourself. It's relatively straight forward.

I would cleanup all of that before start compiling.

**Comment by manhbh** [comment]
By the way, I install box86 from https://github.com/fengxue-jrql/box86-and-box64-for-arm64
I installed wine and *.exe both there are finished. But I cannot launch the file. Do you have any solution to solve this problem? I got an error like this
![image](https://user-images.githubusercontent.com/93500972/181274707-c46e8c3e-85df-49ed-81d2-98ac4a7ea7e1.png)

**Comment by nakele** [comment]
@manhbh congrats first! Also try first something simpler like notepad++

Linux on Proot on android has many limitations including the UDEV error in the console which cannot be fixed (afaik).

Secondly see if someone managed to run that software in wine. Go to winehq in your browser and check there.

Wine version and tricks to make software run usually depends on wine with additional limitations imposed by android.

**Comment by manhbh** [comment]
The notepad++ worked. Awesome.
And the software that I want to launch was tested already on Wine 4.0.2. And I also installed that version
![image](https://user-images.githubusercontent.com/93500972/181288151-c13317c6-e2a8-4c2b-89a0-795d11ff6e53.png)
![image](https://user-images.githubusercontent.com/93500972/181288453-5cb4fe6c-87e5-4ea8-9794-d92b728e820c.png)
